pandas/tests/test_categorical.py

```diff
@@ -1526,6 +1515,46 @@ def test_value_counts(self):
         exp = Series([3,2,1,0], index=["c","b","a","d"])
         tm.assert_series_equal(res, exp)
 
+    def test_value_counts_with_nan(self):
+        # https://github.com/pydata/pandas/issues/9443
+
+        s = pd.Series(["a", "b", "a"], dtype = "category")
```

Comment:
when you are passing arguments, don't use spaces are the equals (e.g. pass as ``dropna=True``)